miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt

```diff
@@ -182,6 +187,10 @@ internal class MiniAppSecureStorageDispatcher(
         clearAllSecureDatabases()
```

Comment:
I dont think `clearAllSecureDatabases()` will work if you try to check `whenReady` status, your `MiniAppSecureStorageDispatcher` initialize when you open a `miniapp`, if you dont open a `miniapp` and then try to clear all the databases, it will not work. I think you should add a exception here if it fails to clear the database.

you should only check `MiniAppSecureStorage` database is available or not, if not create one and delete or just delete it from files without creating one.

Reply:
Well, In fact the `activity` itself won't be initialized. `activity` only will be initialized if a `MiniApp` is opened. Handled all this cases and tested throughly.

Reply:
i think you have `activity` as the hostapp is open. you can use that `context` to remove files from fileSystem.

Reply:
i think you have `activity` as the hostapp is open. you can use that `context` to remove files from fileSystem.

Reply:
Yes, Using `context` only to delete the database.